apps/ledger-live-mobile/src/reducers/accounts.ts

```diff
@@ -4,6 +4,7 @@ import {
   createSelector,
```

Comment:
In this file we implemented a bunch of memoized selectors (made with reselect) that return more granular data than just the basic `accountsSelector`.
It's the base for most of the optimisations made in this PR, as this part of the store is changing at a high frequency for a user with many accounts, so having selectors targeting more specific things is vital to having a good performance in some components by avoiding useless re-renders of these components.